src/line-chart.js

```diff
@@ -106,8 +121,13 @@ class LineChart extends AbstractChart {
             dataset.data
               .map((d, i) => {
                 const x =
-                  paddingRight +
-                  (i * (width - paddingRight)) / dataset.data.length;
+                  i === dataset.data.length - 1 && dataset.data.length !== 1
```

Comment:
this line repeats multiple times in this PR, why is it necessary? you could take it out into its own method to re-use it.